plugins/quick-brick-login-multi-login-providers/index.d.ts

```diff
@@ -46,7 +46,7 @@ declare type LoginData = {
 declare type Logins = LoginData[];
 
 declare type MultiLoginProps = {
-  onCellTap: () => {};
+  onCellTap: (item) => void;
```

Comment:
```suggestion
  onCellTap: (item: ZappEntry) => any;
```
`void` type is not compatible with returning promise (In case onCellTap is async)